bitnami/grafana/templates/service.yaml

```diff
@@ -14,9 +14,12 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
-  {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerIP)) }}
+  {{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges: {{ toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
```

Comment:
As a suggestion, to be able to use a template in this parameter it can be loaded with the `tplValue` fucntion instead of the `toYaml`. That funciton will make a `tpl | toYaml` so it will render the template and then call `tpYaml`. https://github.com/bitnami/charts/blob/c081499a0026ef4fae76be0b4c2b3f5b56a30d55/bitnami/grafana/templates/_helpers.tpl#L262

I can see the example is an IP so maybe there is not any scenario where this can be a template.

Reply:
I looked all over the existing code base and found >10 similar `loadBalancerSourceRanges` and they all just used `toYaml`.